datahub-web-react/src/app/entity/shared/tabs/Documentation/components/LinkList.tsx

```diff
@@ -53,18 +61,124 @@ export const LinkList = ({ refetch }: LinkListProps) => {
         refetch?.();
     };
 
+    const handleEditLink = (metadata: InstitutionalMemoryMetadata) => {
+        form.setFieldsValue({
+            url: metadata.url,
+            label: metadata.description
+        });
+        setLinkDetails(metadata);
+        setEditModalVisible(true);
+    };
+
+    const handleClose = () => {
+        form.resetFields();
+        setEditModalVisible(false);
+    };
+
+    const handleEdit = async (formData: any) => {
+        if (!linkDetails) return;
+
+        try {
+            await removeLinkMutation({
+                variables: { input: { linkUrl: linkDetails.url, resourceUrn: linkDetails.associatedUrn || entityUrn } },
+            });
+
+            if (user?.urn) {
```

Comment:
Why are we checking for user only when adding link?

Reply:
It was just an additional check for the User, I checked that it is not needed as we will always have the user on that screen So I have removed it, Thanks!